src/models/shell.py

```diff
@@ -42,6 +47,7 @@ def do_quit(self, arg) -> bool:
         Quit the game
         """
 
+        "Quit"

```

Comment:
Remove this line please. There probbably went something wrong when resolving conflicts.